pkgs/applications/editors/vscode/vscodium.nix

```diff
@@ -33,6 +33,11 @@ in
 
     sourceRoot = ".";
 
+    postInstall = ''
+      # Install legacy support:
+      ln -s $out/bin/${executableName} $out/bin/vscodium
```

Comment:
Could you explain why we need a backwards-compatible symlink? Does upstream or other linux distributions provide such a symlink?